about.php

```diff
@@ -0,0 +1,211 @@
+                </p>
+            </li> 
+            <li id="li3">
+                <img src="images/eric.jpg"> 
+				<h2 style="position:absolute; left:35%;bottom:1px">zidouemba Eric</h2>
+               <p class="dop-down3 show3">
+                Eric Zidouemba is professor of marketing and management. He has taught in different business schools in Burkina Faso and Senegal. In addition to pursuing his academic interests, he is also active in management and marketing consulting as well as executive and organizational development in Burkina Faso
+                </p>
+            </li> 
+            <li id="li4">
+                <img src="images/isidore.jpeg"> 
+				<h2 style="position:absolute; left:35%;bottom:1px">Kouraogo Isidore</h2>
+               <p class="dop-down4 show4"></p>
+            </li> 
+            <li id="li5">
+                <img src="images/nadia.jpeg"> 
+				<h2 style="position:absolute; left:35%;bottom:1px">Nadia Riedl</h2>
+               <p class="dop-down5 show5">
+                
+                </p>
+            </li> 
+            <li id="li6">
+                <img src="images/desire-copy.jpg"> 
+			<h2 style="position:absolute; left:35%;bottom:1px">Yameogo Desire</h2>
+               <p class="dop-down6 show6"></p>
+            </li> 
+            <li id="li7">
+                <img src="images/gampine.jpg"> 
+				<h2 style="position:absolute; left:35%;bottom:1px"> Gampine Aristide</h2>
+               <p class="dop-down7 show7"> After his master's degree in English at the University of Ouagadougou, Burkina Faso, Aristides Gampine strengthened his abilities in innovative practices of teaching English with Endicott college in Switzerland. he has taught at various international schools and at the American language center. Gampine is pleased that young Burkinabe have the opportunity to take up the challenge of studying entrepreneurship in English at BIT.</p>
+            </li>
+        </ul>
+    </div>
+ <h1>Ours team</h1>
+<p style="text-align:center;width:80%;font-family:Adobe Caslon Pro Bold;margin:auto;opacity:0.6">All of us in the team are studying at Burkina Institute of Technology. To be honest, for us computer science is in the center of our lives we really like it. Knowing that it is very important for every country in the world, our dream is to discover the mystery that is in the back side of the computers. We appeal all the youth to have passion for it. learn it and you will be impressed about what you will find as result. You can also join our team if you love also computer Science and to understand a lot about it <a href="contact.php"><i>Join us</i></a>. For those who are Expert in the domain and want to help us, you are welcome. just contact us.  </p>
+    <script src="js/js.js" type="text/javascript"></script>

```

Comment:
Javascript should be included in the end of the HTML file. Otherwise the browser won't show anything before the script is loaded.